### DISCUSSION TITLE
Milestones for SCI v1

### DISCUSSION THREAD

**Discussion by atg-abhishek** [Green-Software-Foundation/sci#212] [General]
So as we have [now released the alpha version of the SCI](https://greensoftware.foundation/articles/the-green-software-foundation-releases-alpha-version-of-software-carbon-intensity) (!!!), I wanted to discuss the milestones that can help us plan our approach for the work that can do to make a solid v1. Here is what I propose: 

_**For each of the stages, I imagine that it would take 4-6 weeks**_

1. Source and work on 1-2 case studies where we apply the SCI as it stands today
2. Figure out how we want to address reporting of the SCI score 
3. Effective strategy for baselining the SCI itself against future versions 
4. Address gaps identified from the 1-2 case studies in (1) 

Open to suggestions and other items that we might want to include in this work going forward.

**Comment by SaraEmilyBergman** [comment]
Something that I have been thinking of that could be interesting is to try and find only the software boundary and the R of a wide selection of different types of software. 
I think that; 1) it can be easier to get access too (thinking of NDAs and anti-trust policies and general need for confidentiality of some companies) and 2) it can give us an idea of potential flaws and 3) give us a longer list (perhaps as an appendix) of software boundary and R for v1 of the specification.

**Comment by Henry-WattTime** [comment]
Case study potentials:
-Microsoft (Will) green AI training, writing up outline
-Shell (Adam) strong interest
-Goldman Sachs (Sarah)? -infrastructure
-Accenture (Navveen) - identifying case study

Baselining:
-as the SCI evolves, in the interest of transparency, nice to have a back test to see how case studies would change
-changes, made improvements, baseline SCI not applications, see how score changes.
-Implement at first major specification update?

**Reply by atg-abhishek** [reply]
@buchananwp will make an issue with his use-case and @navveenb will share updates on the internal use case at Accenture.

**Comment by Sealjay** [comment]
> 
> 2. Figure out how we want to address reporting of the SCI score 
> 

For milestone two, I can see the PR has been merged for the requirements, and I've opened an issue to propose a specific reporting project:

https://github.com/Green-Software-Foundation/standards_wg/issues/2
